src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -20,10 +20,10 @@ if (PHP_VERSION_ID >= 70200) {
     $PHPUNIT_VERSION = getenv('SYMFONY_PHPUNIT_VERSION') ?: '6.5';
 } elseif (PHP_VERSION_ID >= 50600) {
     // PHPUnit 4 does not support PHP 7
-    $PHPUNIT_VERSION = getenv('SYMFONY_PHPUNIT_VERSION') ?: '5.7';
+    $PHPUNIT_VERSION = getenv('SYMFONY_PHPUNIT_VERSION') ?: '5.7.26';
```

Comment:
6.5, 5.7 and 4.8 seem to be aliases for the latest patch versions. We can't fix it to a specific patch version (problem to maintain it). And maybe 4.8 and 5.7 aliases will be available again soon. What about use a fallback if the zip is not found and using github api for selecting the latest patch tag ?

Reply:
Let's try saving us from any additional complexity.
Let's lobby https://github.com/sebastianbergmann/phpunit/issues/3413